lib/flipper.rb

```diff
@@ -55,19 +67,19 @@ def self.group_exists?(name)
   # Returns the Flipper::Group if group registered.
   # Raises Flipper::GroupNotRegistered if group is not registered.
   def self.group(name)
-    groups.get(name)
+    groups_registry.get(name)
   rescue Flipper::Registry::KeyNotFound => e
     raise GroupNotRegistered, "Group #{e.key.inspect} has not been registered"
   end
 
-  # Internal: Registry of all groups.
```

Comment:
I always hate this part. Some stuff needs to be tested, but I don't want to support it publicly. I'm open to suggestions.